build from source releases fails due to missing git directory

**Describe the bug**
build from source releases fails due to missing git directory

**To Reproduce**
Steps to reproduce the behavior:
1. Download release source (ie:  wget https://github.com/OpenRefine/OpenRefine/archive/3.1.tar.gz)
1. Run `./refine build`

**Current Results**
```
[ERROR] Failed to execute goal pl.project13.maven:git-commit-id-plugin:2.2.4:revision (get-the-git-infos) on project main: .git directory is not found! Please specify a valid [dotGitDirectory] in your pom.xml -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :main
Error: Error while running maven task 'process-resources'
The command '/bin/sh -c ./refine build' returned a non-zero code: 1
```

**Expected behavior**
A clean build

**Desktop (please complete the following information):**
 - OS: linux (using latest `alpine` docker image)
 - JRE or JDK Version:
```
java -version
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (IcedTea 3.12.0) (Alpine 8.212.04-r0)
OpenJDK 64-Bit Server VM (build 25.212-b04, mixed mode)
```

**OpenRefine (please complete the following information):**
 - Version: OpenRefine 3.1

**Additional context**
Trying to build docker images with a few extensions. Upstream images would be appreciated :D

--- a/main/pom.xml
+++ b/main/pom.xml
@@ -247,6 +247,7 @@
             <verbose>false</verbose>
             <generateGitPropertiesFile>true</generateGitPropertiesFile>
             <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
+            <failOnNoGitDirectory>false</failOnNoGitDirectory>
             <format>json</format>
             <gitDescribe>
                 <skip>false</skip>

